GitUI/MessageBoxes.cs

```diff
@@ -124,13 +134,116 @@ public static bool ConfirmUpdateSubmodules(IWin32Window win)
         public static void ShellNotFound([CanBeNull] IWin32Window owner)
             => ShowError(owner, Instance._shellNotFound.Text, Instance._shellNotFoundCaption.Text);
 
+        public static void Show([CanBeNull] IWin32Window owner, ExternalOperationException ex, ExternalOperationExceptionFactory.Handling exceptionHandling)
+        {
+            if (exceptionHandling == ExternalOperationExceptionFactory.Handling.None)
+            {
+                return;
+            }
+
```

Comment:
Why do we need to use "IWin32Window owner".
Most likely this question applies to all methods in this class, but this one in particular. 
Are there any cases where the current window will not own the message box?

Reply:
IIRC there are cases where owner is a different window.

Reply:
Do you know specific cases?